modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -292,11 +276,17 @@ def __run(self, cmd):
 
     def __form_cmd(self, node, cmd):
         ssl = ""
-        if hasattr(self, 'key_store_path'):
-            ssl = f" --keystore {self.key_store_path} --keystore-password {self.key_store_password} " \
-                  f"--truststore {self.trust_store_path} --truststore-password {self.trust_store_password}"
-
-        return self._cluster.script(f"{self.BASE_COMMAND} --host {node.account.externally_routable_ip} {cmd} {ssl}")
+        if hasattr(self.creds, 'key_store_path'):
```

Comment:
As it self.creds is a class now, then it does have 'key_store_path' attr, doesn't it? Should it to be replaced with check for None?